### ISSUE TITLE
skip references to branches

### ISSUE THREAD

**Issue by vchepkov** [camptocamp/puppetfile-updater#16]
when Puppetfile has reference to branch, I don't think any modifications are required or at least should be optional. For example:

```
 mod 'r10k',
   :git    => 'https://github.com/vchepkov/puppet-r10k.git',
   :branch => 'upgrade5'
```

gets transformed to 

```
 mod 'r10k',
   :git    => 'https://github.com/vchepkov/puppet-r10k.git',
   :branch => 'upgrade5', :ref => "c532c57"
```

**Comment by raphink** [comment]
puppetfile-updater currently ignores branch settings, so the example you gave is seen as a reference to the master branch. While this is probably not what is expected, it is generally safer to point to fixed points (exact forge versions, tags, commits) instead of moving targets (semver moving references or branches).

I'm open to supporting branches if you submit a PR though.

**Comment by vchepkov** [comment]
Would skipping github modules altogether be an acceptable workaround?
I submitted #18 if it is

**Comment by raphink** [comment]
fixed in #18
